workspace_host/interface.sql

```diff
@@ -145,7 +145,8 @@ UPDATE
 SET
     state = 'unhealthy',
     state_changed_at = NOW(),
-    unhealthy_at = NOW()
+    unhealthy_at = NOW(),
+    unhealthy_reason = $unhealthy_reason
 WHERE
     wh.instance_id = $instance_id
     AND wh.state IN ('launching', 'ready', 'draining');
```

Comment:
```suggestion
    AND wh.state != 'unhealthy';
```
Not sure which one we want here (current code ignores `terminating` and `terminated`).

Reply:
I think we want to stick with the current version. The `terminating` and `terminated` states should be a one-way trip to oblivion 😄